rollup.config.js

```diff
@@ -108,18 +108,22 @@ function createCommonJSConfig(input, output, options) {
 }
 
 function createUMDConfig(input, output, env) {
-  const c = output.split('/').pop()
+  const c = output.replace(/^dist\/umd\//, '').split(/\W/)
```

Comment:
I think using a longer name than `c` will be beneficial, because right now it's hard to understand.

Reply:
Yeah, I agree. How would you name instead of `c` if you have an idea?

Reply:
I don't know, because I don't know the _intent_. `outputConfigPath`? I'm only guessing from the usage, but I don't know what did you mean.

You know, there are two things hard in programming: cache invalidation and naming things ;)

Reply:
Yeah, I don't know what to name it. The intent is, output file name parts divided by non words.
Maybe there's a better solution with regex, which could be harder to read.

Reply:
My best guess then is `outputFilenameWords`, but it won't force it